apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-welcome-tour/src/store.js

```diff
@@ -2,10 +2,31 @@
  * External dependencies
  */
 import apiFetch from '@wordpress/api-fetch';
-import { registerStore } from '@wordpress/data';
+import { combineReducers, registerStore } from '@wordpress/data';
 
-const reducer = ( state = {}, { type, isNuxEnabled } ) =>
-	'WPCOM_BLOCK_EDITOR_NUX_SET_STATUS' === type ? { ...state, isNuxEnabled } : state;
+const isNuxEnabledReducer = ( state = null, action ) => {
```

Comment:
I have a feeling the default value of the state here should be `undefined`.

We're doing a type check over in [tour-launch.js](https://github.com/Automattic/wp-calypso/blob/fd46f55c4296c4c859c9cff582a2bc5692f0d16e/apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-welcome-tour/src/tour-launch.js#L35) that explicitly checks for `undefined`. 

But if the default state value is `null`, it will never pass the test.

I noticed this when testing, and clearing my local storage/IndexedDB while sandboxed to force the tour to reopen.

Reply:
Yes....you are right. 👍